### PR TITLE
fixed an issue with gmail smtp. Included from_email in message

### PR DESCRIPTION
**Pull request by hassanabidpk**
I am using gmail smtp - Although I have setup from_email (get from the form) but still from email is gmail used for setting up smtp. Therefore I appended it in message. Please review or suggest alternative.

### CODE REVIEW

File: landingsite/views.py [hunk 1]
```diff
@@ -7,11 +7,11 @@ def index(request):
     if request.method == "POST":
```
Comment (jinpark) [inline landingsite/views.py]: this might not be in the scope of this pr but not sure why we have a post going to the index page. we should probably have a specific url and view for receiving email forms and just let the root url always be a get request

File: landingsite/views.py [hunk 2]
```diff
@@ -7,11 +7,11 @@ def index(request):
...
         message = request.POST.get('message', '')
-        print (message)
         from_email = request.POST.get('email', '')
         subject = 'Get in touch - Code For Everyone website'
         if subject and message and from_email and full_name:
             try:
+                message += "\n From: " + from_email
                 send_mail(subject, message, from_email, ['info@codeforeveryone.co'])
             except BadHeaderError:
                 return HttpResponse('Invalid header found.')
```
Comment (jinpark) [inline landingsite/views.py]: Can we add some logging here? Then we could keep track of how many times we're having this error

File: landingsite/views.py [hunk 3]
```diff
@@ -7,11 +7,11 @@ def index(request):
     if request.method == "POST":
         full_name = request.POST.get('full_name', '')
         message = request.POST.get('message', '')
-        print (message)
         from_email = request.POST.get('email', '')
         subject = 'Get in touch - Code For Everyone website'
         if subject and message and from_email and full_name:
             try:
+                message += "\n From: " + from_email
```
Comment (jinpark) [inline landingsite/views.py]: This change looks fine to me. I'm not sure how to test it though?

File: landingsite/views.py [hunk 4]
```diff
@@ -7,11 +7,11 @@ def index(request):
     if request.method == "POST":
         full_name = request.POST.get('full_name', '')
         message = request.POST.get('message', '')
-        print (message)
         from_email = request.POST.get('email', '')
```
Comment (jinpark) [inline landingsite/views.py]: We should probably error out here if we dont have an email because whats the point of them sending us an email if we dont know who its from.

Also, we should handle client side validation

**Comment by jinpark** [comment]
Overall, the change you did looks fine so feel free to merge but we should probably fix some of the problems I mentioned here

File: landingsite/views.py (hunk 1 above)
Comment (hassanabidpk) [inline landingsite/views.py]: Okay. I will make it different url.

File: landingsite/views.py (hunk 3 above)
Comment (hassanabidpk) [inline landingsite/views.py]: Codeforeveryone.co/contact-us is the only way to check it. Recently Google has made SMTP more secure by including app specific passwords (2 factor authentication).

File: landingsite/views.py (hunk 4 above)
Comment (hassanabidpk) [inline landingsite/views.py]: Yup. I will do separate PR for it

File: landingsite/views.py (hunk 2 above)
Comment (hassanabidpk) [inline landingsite/views.py]: Server logging ? How could we do it?

File: landingsite/views.py (hunk 1 above)
Comment (hassanabidpk) [inline landingsite/views.py]: Will do separate PR for the all issues mentioned.

File: landingsite/views.py (hunk 2 above)
Comment (jinpark) [inline landingsite/views.py]: https://docs.djangoproject.com/en/1.8/topics/logging/

We can add this in the next pr as you stated before

**Comment by jinpark** [comment]
Just to reiterate, feel free to merge, just dont forget to make a issues/prs for the things I commented on :)

**Comment by hassanabidpk** [comment]
Merged #18
